### PR TITLE
Add addon Quote Code Beautifier

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
This adds an addon based on @BoomerScratch's Scratch Forums Quote Code Beautifier.

### CODE REVIEW

**Comment by Hans5958** [comment]
Name capitalization?

File: addons/forum-quote-code-beautifier/addon.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Forums Quote & Code Beautifier",
```
Comment (Hans5958) [inline addons/forum-quote-code-beautifier/addon.json:2]: Name capitalization?

**Review by Hans5958 — CHANGES_REQUESTED** [review]

File: addons/forum-quote-code-beautifier/addon.json (hunk 1 above)
Comment (mxmou) [reply addons/forum-quote-code-beautifier/addon.json:2]: Also, a name like "Customized quotes on forums" would better explain this addon's purpose.

Comment (Hans5958) [reply addons/forum-quote-code-beautifier/addon.json]: Well, I think the name should be ok the short side than the descriptive side. It's just the name, not the description.

@GrahamSH-LLK sorry for late response on this one

**Review by mxmou — APPROVED** [review]

**Review by WorldLanguages — CHANGES_REQUESTED** [review]
Please make this not recommended and disabled by default

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**